<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDokter</class>
 <widget class="QMainWindow" name="FrmDokter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormDokter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DOKTER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>581</width>
      <height>392</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>40</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="idDokterLabel">
       <property name="text">
        <string>IdDokter</string>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Txt_IdDokter"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="namaLabel">
       <property name="text">
        <string>Nama</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Txt_Nama"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="bidangLabel">
       <property name="text">
        <string>Bidang</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="Cmb_Bidang">
       <item>
        <property name="text">
         <string>Dokter Umum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kandungan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Anak</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Penyakit Dalam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bedah</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="jadwalLabel">
       <property name="text">
        <string>Jadwal</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="Chk_Senin">
           <property name="text">
            <string>Senin</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Jam Buka</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time1_Buka">
             <property name="dateTime">
              <datetime>
               <hour>8</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>2</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Jam Tutup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time1_Tutup">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="Chk_Selasa">
           <property name="text">
            <string>Selasa</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Jam Buka</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time2_Buka">
             <property name="dateTime">
              <datetime>
               <hour>8</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Jam Tutup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time2_Tutup">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="Chk_Rabu">
           <property name="text">
            <string>Rabu</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Jam Buka</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time3_Buka">
             <property name="dateTime">
              <datetime>
               <hour>8</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Jam Tutup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time3_Tutup">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QCheckBox" name="Chk_Kamis">
           <property name="text">
            <string>Kamis</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Jam Buka</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time4_Buka">
             <property name="dateTime">
              <datetime>
               <hour>8</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Jam Tutup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time4_Tutup">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QCheckBox" name="Chk_Jumat">
           <property name="text">
            <string>Jumat</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Jam Buka</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time5_Buka">
             <property name="dateTime">
              <datetime>
               <hour>8</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Jam Tutup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time5_Tutup">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QCheckBox" name="Chk_Sabtu">
           <property name="text">
            <string>Sabtu</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Jam Buka</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time6_Buka">
             <property name="dateTime">
              <datetime>
               <hour>8</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Jam Tutup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time6_Tutup">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QCheckBox" name="Chk_Minggu">
           <property name="text">
            <string>Minggu</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Jam Buka</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time7_Buka">
             <property name="dateTime">
              <datetime>
               <hour>8</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Jam Tutup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="Time7_Tutup">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>h:mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="kapasitasLabel">
       <property name="text">
        <string>Kapasitas</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>300</number>
       </property>
       <item>
        <widget class="QLineEdit" name="Txt_Kapasitas"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>orang / jam</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>Phone</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Txt_Phone"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>293</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <widget class="QPushButton" name="PB_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_update">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_del">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
